scss/civicrm/common/_modals.scss

```diff
@@ -39,7 +38,7 @@
     border-bottom: 1px solid $crm-dark-grayblue !important;
     box-shadow: $crm-contribute-form-shadow;
     color: $gray-darker !important;
-    padding: 15px 20px;
+    padding: 13px 20px; // @TODO: use modal header padding variable
```

Comment:
Any reason why we don't want to make it a variable like `$modal-block-padding` right now?

Reply:
why is a TODO at all? what's preventing from doing it now?

Reply:
@igorpavlov because it's only used once.

Reply:
@AkA84 this modal is from jQuery UI, other modals use Bootstrap UI (specially on SSP). If the [$modal-title-padding](https://github.com/compucorp/org.civicrm.shoreditch/blob/staging/scss/bootstrap/overrides/_variables.scss#L202) variable is used and modified, it would affect other modals. 

This is probably desired since all modals should look the same regardless of their source, but I fear that would stretch the goal for this ticket even further.

Reply:
@igorpavlov I'll add the values to a variable. It would explain better the 13px.

Reply:
Understood @Rene. I think in case of CSS we might want to follow the rule: if there is a style in mockups (invision) it should go to a variable. There are some style not needing a variable, I am talking about some "system-like" styles like "display: block". But padding `13px 20px` is something that the mockup explicitly dictates. cc @AkA84

Reply:
@reneolivo 

> If the $modal-title-padding variable is used and modified, it would affect other modals.

Absolutely true. Then it should be smth like

```css
/* UI-Modals */

.ui-modal-header-padding {
  /* ... */
}
```

Reply:
@reneolivo sorry, I meant a variable, not a class selector:

```css
/* UI-Modals */
$ui-modal-header-padding: smth;
```